doc/source/examples/03_modeling/design_parameters.mystnb

```diff
@@ -0,0 +1,145 @@
+---
+jupytext:
+  text_representation:
+    extension: .mystnb
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.4
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Modeling: Using Design Parameters
+
+You can read and update parameters that are part of the design.
+In this example, we have a simple design with two associated parameters.
+
++++
+
+## Perform required imports
+
+```{code-cell} ipython3
+import os
+import requests
+from ansys.geometry.core import launch_modeler
+from ansys.geometry.core.modeler import *
+from ansys.geometry.core.parameters import *
+```
+
+The file for this example is in the integration tests folder, so let's download it.
+
++++
+
+## Download the example file
+
++++
+
+For this example, download the file from the pyansys-geometry repository.
+
+```{code-cell} ipython3
+import requests
+
+def download_file(url, filename):
+    """Download a file from a URL and save it to a local file."""
+    response = requests.get(url)
+    response.raise_for_status()  # Check if the request was successful
+    with open(filename, 'wb') as file:
+        file.write(response.content)
+
+# URL of the file to download
+url = "https://github.com/ansys/pyansys-geometry/blob/main/tests/integration/files/blockswithparameters.dsco"
+
+# Local path where the file will be saved
+file_name = "blockswithparameters.dsco"
+current_path = os.getcwd()
+file_path = os.path.join(current_path, file_name)
+# Download the file
+download_file(url, file_path)
+print("File is downloaded to " + file_path)
+```
+
+## Import a design with parameters
+
++++
+
+Importing the model using ``open_file`` method of the modeler.
+
+```{code-cell} ipython3
+# Create a modeler object
+modeler = launch_modeler()
+design = modeler.open_file(file_path)
+design.plot()
+```
+
+## Read existing parameters of the design
+
+You can get all the parameters of the design as a list of parameters. In this example there is two parameters. So we can see the two items in the list.
+
+```{code-cell} ipython3
+my_parameters = design.get_all_parameters()
+print(len(my_parameters))
+```
+
+A parameter object has a name, value, and unit.
+
+```{code-cell} ipython3
+print(my_parameters[0].name)
+print(my_parameters[0].dimension_value)
+print(my_parameters[0].dimension_type)
+
+print(my_parameters[1].name)
+print(my_parameters[1].dimension_value)
+print(my_parameters[1].dimension_type)
+```
+
+Parameter values are returned in the default unit for each dimension type. Since default length unit is meter and default area unit is meter square, the value is returned in metersquare.
+
++++
+
+## Edit a parameter value
+
+You can edit the parameters name or value by simply setting these fields. Let's
+set the second parameter (p2 value to 350 mm. )
```

Comment:
```suggestion
You can edit a parameter's name or value by simply setting these fields. This code
sets the second parameter (p2 value) to 350 mm. 
```
I see parameter1 being set to 0.000440, but that's not what the text says. Is an update to the next needed?